python/private/pypi/extension.bzl

```diff
@@ -863,6 +826,40 @@ This is only used if the {envvar}`RULES_PYTHON_ENABLE_PIPSTAR` is enabled.
 """,
     ),
     # The values for PEP508 env marker evaluation during the lock file parsing
+    "os_name": attr.string(
+        doc = """\
+The OS name to be used.
```

Comment:
As a user, how do I know what value to use? e.g. Linux, linux, macos, MacOS, OSX, windows, Windows, Win, etc etc

I think this is the doc? https://packaging.python.org/en/latest/specifications/dependency-specifiers/#environment-markers

Reply:
The `os_name` is just tho `@platforms//os:<name>`. For env markers, I think we could do our best to add the documentation from our internal comments.